statefun-flink/statefun-flink-core/src/main/java/org/apache/flink/statefun/flink/core/httpfn/OkHttpUtils.java

```diff
@@ -38,6 +41,9 @@ static OkHttpClient newClient() {
         .followRedirects(true)
         .followSslRedirects(true)
         .retryOnConnectionFailure(true)
+        // Enable HTTP/2 if available (uses H2 upgrade),
+        // otherwise fallback to HTTP/1.1
+        .protocols(Collections.singletonList(Protocol.HTTP_2))
```

Comment:
It seems like okhttp has HTTP_2  already in the default protocol list, so this is not needed. 
Moreover the above code throws an exception (when specifying HTTP_2 you must also specify HTTP_1) but if you would look at the default builder you would see that okhttp already has HTTP_2 and HTTP_1 in the list of default protocols.

As an attempt to "make it work" I've artificially added the prefix `http://localhost/` to the endpoint, what do you think about that (it is technically "localhost")?

Reply:
Ah i see:

```
  static final List<Protocol> DEFAULT_PROTOCOLS = Util.immutableList(
      Protocol.HTTP_2, Protocol.HTTP_1_1);
```

I'm gonna remove this